sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -339,6 +339,21 @@ public void close() {
             ((Closeable) integration).close();
           }
         }
+
+        withScope(
+            scope -> {
+              ITransaction transaction = scope.getTransaction();
+              if (transaction != null) {
+                for (Span span : transaction.getSpans()) {
+                  span.setSpanFinishedCallback(null);
+                  span.finish(SpanStatus.CANCELLED);
+                }
+                transaction.finish(SpanStatus.CANCELLED);
```

Comment:
You can use `ITransaction.forceFinish()` here instead as it will basically do the same.

Reply:
Always learning new things 😅